### ISSUE TITLE
Feature request: automatically change tournament viewed game

### ISSUE THREAD

**Issue by Baikalcadillac** [veloce/lichobile#1305]
Right now if you are going to active tournament, then scroll down to the current game, after game ends nothing happens. I think it would be good to automatically switch to the other active game in current tournament after previous game is ended.

**Comment by freefal** [comment]
This works fine for me. Can you try a few tournaments and make sure this
happens consistently for you. If it does, can you provide details of the
app and OS version you're using?

**Comment by Baikalcadillac** [comment]
I'm talking about viewing tournament games while not participating in it. Tried different variants and time controls, but still the same - game ends and nothing happens. 
Lichess 7.2.0, iOS 12.4.7

**Comment by freefal** [comment]
I am on the same and have no issue. Could you check with a bullet
tournament? If you're looking at a longer time control tournament, maybe it
takes a little while for a new game to start? The current expected behavior
is to to switch to other active games already in progress.

**Comment by Baikalcadillac** [comment]
I've tried again couple of times with bullet and blitz tournaments, both browser and app version doesn't switch automatically. However preview of the current playing game changes after game was ended.

**Comment by freefal** [comment]
Got it. You are looking for a sort of Tournament TV, where if you click on
the current game on the tournament page, the next game pops up
automatically like when you watch, for example, Blitz TV on the app or
website. This doesn't exist today, but we can leave this issue open as I
agree it would be useful.

**Comment by Baikalcadillac** [comment]
Yes, that's exactly what I was meant. Sorry for my confusing phrasing.
I was pretty sure this feature already exists in browser version, thanks for clarifying.
